packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -39,41 +43,64 @@ export class AtomicSearchInterface {
   @Prop() sample = false;
   @Prop({reflect: true}) pipeline = 'default';
   @Prop({reflect: true}) searchHub = 'default';
-  @Prop() logLevel?: LogLevel = 'silent';
+  @Prop() logLevel?: LogLevel;
   @Prop() i18n: i18n = i18next.createInstance();
+  @Prop({reflect: true}) language = 'en'; // TODO: make watchable and update i18next language on change
   @Prop({mutable: true}) engine?: Engine;
   @State() error?: Error;
+  @Event() ready!: EventEmitter;
```

Comment:
Open to another name, but having an event (or callbacks like before) is necessary, property like engine/i18n aren't available right when the custom element is defined

Reply:
Ready seems good to me 👍